recipes/microtar/all/conanfile.py

```diff
@@ -8,52 +12,59 @@ class MicrotarConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/rxi/microtar"
     description = "A lightweight tar library written in ANSI C"
-    topics = ("conan", "tar", "archive")
-    settings = "os", "arch", "compiler", "build_type"
-    options = {"shared": [True, False],
-               "fPIC": [True, False]}
-    default_options = {"shared": False,
-                       "fPIC": True}
-    exports_sources = ["CMakeLists.txt"]
-    generators = "cmake",
-    _cmake = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    topics = ("tar", "archive")
 
-    def source(self):
-        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
 
-    def configure(self):
-        if self.options.shared:
-            del self.options.fPIC
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+    exports_sources = "CMakeLists.txt"
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
-    def _configure_cmake(self):
-        if self._cmake:
-            return self._cmake
-        cmake = CMake(self)
-        cmake.configure(build_folder=self._build_subfolder)
-        self._cmake = cmake
-        return self._cmake
+    def configure(self):
+        if self.options.shared:
+            try:
+                del self.options.fPIC
```

Comment:
Question for a curiosity reasons, why not `rm_safe`?

Reply:
it requires conan 1.53.0 and c3i still runs conan 1.52.0